web/README.md

```diff
@@ -3,3 +3,75 @@
 This is the main front-end web site project. This project will consume the output of the front end components project via NPM.
 
 This is an MVC web app written in C#. It's main purpuse is to provide proxy authentication/auuthorization services, alomng with any static pages that need serving
+
+## Getting started
+
+### Run the Web App locally
+
+#### Secret storage
+In a console window:
+1. Navigate to `EducationBenchmarking.Web` project root
+2. Run `dotnet user-secrets init` to initialise secrets in the directory
+3. Run the follow to create XXXX secret: `dotnet user-secrets set "XXXX" "xxxxx"`
+
+#### Platform APIs
+If running the Platform APIs locally then no further required, just ensure the API port configuration matches that in `appsettings.Development.json`.
+
+However if you are using deployed instances of the Platform APIs then having initialised secret storage, in a console window:
+1. Navigate to `EducationBenchmarking.Web` project root
+2. Set Insight API url user secret: `dotnet user-secrets set "Apis:Insight:Url" "[INSERT URL VALUE]"`
+3. Set Insight API key user secret: `dotnet user-secrets set "Apis:Insight:Key" "[INSERT KEY VALUE]"`
+4. Set Benchmark API url user secret: `dotnet user-secrets set "Apis:Benchmark:Url" "[INSERT URL VALUE]"`
+5. Set Benchmark API key user secret: `dotnet user-secrets set "Apis:Benchmark:Key" "[INSERT KEY VALUE]"`
+6. Set Establishment API url user secret: `dotnet user-secrets set "Apis:Establishment:Url" "[INSERT URL VALUE]"`
+7. Set Establishment API key user secret: `dotnet user-secrets set "Apis:Establishment:Key" "[INSERT KEY VALUE]"`
+
+#### DfE Sign-in (DSI) authentication
+Having initialised secret storage, in a console window:
```

Comment:
It would be handy to define the secret keys, as above. e.g. `"DFESignInSettings:APISecret"`

Reply:
Agree - left them out of the moment, as it's non critical.